cs-config/install.sh

```diff
@@ -2,5 +2,15 @@
 
 git clone https://github.com/PSLmodels/OG-USA
 cd OG-USA
-conda install scipy mkl PSLmodels::taxcalc conda-forge::paramtools
+# make sure the git object is up to date, in case
+# it's cached.
+git fetch origin
+git merge origin/master
```

Comment:
@jdebacker You can delete this now. I was having an issue with caching a few weeks ago, but things seem to be working better now. These commands will increase the size of the git data.

Reply:
@hdoupe Ok - deleted.